### PR TITLE
docs: update README about section

### PR DESCRIPTION
**Pull request by anguillanneuf**

### CODE REVIEW

**Review by eaball35 — APPROVED** [review]

**Comment by anguillanneuf** [comment]
@eaball35 This PR is still in a draft form :-)

File: .readme-partials.yaml [hunk 1]
```diff
@@ -97,3 +97,14 @@ custom_content: |
...
+about: |
+  [Google Cloud Pub/Sub Lite][product-docs] is a zonal, real-time messaging
+  service that lets you send and receive messages between independent
+  applications. You can manually configure the throughput and storage capacity
+  for Pub/Sub Lite systems.
+
+  The Pub/Sub Lite Spark connector supports Pub/Sub Lite as an input source to
+  Apache Spark Structured Streaming in both the default micro-batch processing
+  mode and the _experimental_ continous processing mode. The connector works in
+  all Apache Spark distributions, including [Google Cloud Dataproc](https://cloud.google.com/dataproc/docs/), Databricks,
```
Comment (jiangmichaellll) [inline .readme-partials.yaml:109]: Let's remove Databricks for now. This should only be added after the announcement.

File: README.md [hunk 2]
```diff
@@ -50,29 +50,22 @@ You will need to [enable billing][enable-billing] to use Google Pub/Sub Lite Spa
...
-[Pub/Sub Lite Spark Connector][product-docs] is designed to provide reliable,
-many-to-many, asynchronous messaging between applications. Publisher
-applications can send messages to a topic and other applications can
-subscribe to that topic to receive the messages. By decoupling senders and
-receivers, Google Cloud Pub/Sub allows developers to communicate between
-independently written applications.
+The Pub/Sub Lite Spark connector supports Pub/Sub Lite as an input source to
+Apache Spark Structured Streaming in both the default micro-batch processing
+mode and the _experimental_ continous processing mode. The connector works in
+all Apache Spark distributions, including [Google Cloud Dataproc](https://cloud.google.com/dataproc/docs/), Databricks,
```
Comment (jiangmichaellll) [inline README.md:66]: same to Databricks here.

**Review by jiangmichaellll — APPROVED** [review]

File: .readme-partials.yaml (hunk 1 above)
Comment (anguillanneuf) [reply .readme-partials.yaml]: Done.

**Pull request merged**